lib/mix/lib/mix/tasks/test.ex

```diff
@@ -17,14 +19,64 @@ defmodule Mix.Tasks.Test do
       output = opts[:output]
 
       fn ->
-        Mix.shell().info("\nGenerating cover results ...")
         File.mkdir_p!(output)
 
-        Enum.each(:cover.modules(), fn mod ->
-          {:ok, _} = :cover.analyse_to_file(mod, '#{output}/#{mod}.html', [:html])
-        end)
+        Mix.shell().info("\nCover results")
```

Comment:
We can keep this line as it was before because it will also work if folks choose to skip the summary.

Reply:
How about keeping this line and adding additional output after HTML result was generated?